x/tx/signing/aminojson/time.go

```diff
@@ -46,7 +46,7 @@ func marshalTimestamp(message protoreflect.Message, writer io.Writer) error {
 
 // MaxDurationSeconds the maximum number of seconds (when expressed as nanoseconds) which can fit in an int64.
 // gogoproto encodes google.protobuf.Duration as a time.Duration, which is 64-bit signed integer.
-const MaxDurationSeconds = int64(math.MaxInt64/int(1e9)) - 1
+const MaxDurationSeconds = int64(math.MaxInt64)/1e9 - 1
```

Comment:
Since `math.MaxInt64` is an untyped constant, an explicit cast is required to support 32 arch, e.g. our arm docker builds.   See https://github.com/golang/go/issues/23086